packages/migration/migrations/hearth/20220822085901-rejected-declarations-started-by.ts

```diff
@@ -114,7 +125,9 @@ function getTaskCursor(db, compositionIds) {
     }
   }
   const sort = { $sort: { 'meta.lastUpdated': 1 } }
-  return db.collection('Task_history').aggregate([query, projection, sort])
+  return db
+    .collection('Task_history')
+    .aggregate([query, projection, sort]) as unknown as fhir.Task
```

Comment:
This doesn't really return a fhir.Task, instead it should be a cursor of fhir.Task